cli/schemas/config-file.json

```diff
@@ -0,0 +1,542 @@
+{
+  "$id": "https://deno.land/schemas/config-file.json",
```

Comment:
Maybe we should version this url?

Reply:
How do you propose to do that?

Reply:
I guess maybe it doesn't matter actually if we're going to be providing this file text to the editor plugins via the LSP (not sure how that's going to work atm).

Reply:
I think we should version it at least, it is as simple as adding a version to the file.  I suggest we use something like `config-file_v1.json` and simply increment the number when we make changes, versus a full blown semver scheme.  Then in vscode you associate a schema with a file type, and we will just bump it when logical.  The other option is we just let git/GitHub manage it, with the raw github with a commitish path be the id we use in vscode to "pin" it.